app/assets/javascripts/discourse/app/components/sidebar/section.hbs

```diff
@@ -36,9 +36,9 @@
     </div>
 
     {{#if this.displaySectionContent}}
-      <div class="sidebar-section-content">
+      <ul class="sidebar-section-content" id={{this.sidebarSectionID}}>
```

Comment:
```suggestion
      <ul class="sidebar-section-content" id={{this.sidebarSectionContentID}}>
```

I don't think `sidebarSectionID` is quite right here because sidebar section generally refers to the entire section including the header.